src/components/navigation/private/index.js

```diff
@@ -0,0 +1,8 @@
+import React, { Fragment } from "react"
+import { Link } from "@reach/router";
+
+export const NavPrivate = () =>
+  <Fragment>
+    <Link to='/app/profile'>Profile</Link>
```

Comment:
Write a an iterator function that will build this jsx when provided with an array of objects with all the desired routes config. Make sure to add `enabled: true` property to switch any route off/on for better usage.